Automattic/wp-calypso
# Performance Profiler: Use png for example report image in banner

Related to https://github.com/Automattic/wp-calypso/pull/94585#issuecomment-2357820378

## Proposed Changes

* Use PNG for the the email report example image in the banner

|Size 1 | Size 2|Size 3 | Size 4|Size 5|
|-------|------|------|------|------|
|      ![CleanShot 2024-09-18 at 12 14 32@2x](https://github.com/user-attachments/assets/c6d4f5e0-568b-4f6c-b048-65faf63f0dde)    |       ![CleanShot 2024-09-18 at 12 14 46@2x](https://github.com/user-attachments/assets/7c433415-cab2-49a2-ba7f-8033ee5d4953)   |       ![CleanShot 2024-09-18 at 12 15 00@2x](https://github.com/user-attachments/assets/e4488629-199b-48ec-916c-60c446a8bebc)     |          ![CleanShot 2024-09-18 at 12 15 11@2x](https://github.com/user-attachments/assets/fa55f132-3033-412b-9210-47f6741b532d) |          ![CleanShot 2024-09-18 at 12 15 20@2x](https://github.com/user-attachments/assets/b3ffbbff-1d94-41b4-ad49-4b0eb2d6b052)    |

## Why are these changes being made?


* To improve the quality of the visuals

## Testing Instructions



* Open Profiler Tool and check the banner
* Check the screenshot image displays nicely in the banner

## Pre-merge Checklist
